administrator/language/de-DE/de-DE.com_installer.ini

```diff
@@ -179,25 +179,25 @@ COM_INSTALLER_MSG_WARNINGS_SMALLPOSTSIZE="Niedrige PHP-Maximum-POST-Größe"
 COM_INSTALLER_MSG_WARNINGS_SMALLUPLOADSIZE="Maximale PHP-Dateihochladegröße zu klein: Sowohl der Wert für „upload_max_filesize“, als auch für „post_max_size“ muss in der php.ini und/oder der .htaccess-Datei definiert werden."
 COM_INSTALLER_MSG_WARNINGS_SMALLUPLOADSIZEDESC="Die maximale Dateigröße für das Hochladen von Dateien („upload_max_filesize“) liegt unter 8 MB. Dies kann zu Problemen bei der Installation von großen Erweiterungen führen."
-COM_INSTALLER_MSG_WARNINGS_UPDATE_NOTICE="Vor der Aktualisierung ist zu prüfen, dass diese auch kompatibel mit der eingesetzten Joomla!-Installation ist. <br>Es wird dringend empfohlen, vor der Aktualisierung ein <strong>Backup</strong> der Installation zu erstellen."
+COM_INSTALLER_MSG_WARNINGS_UPDATE_NOTICE="Vor dem Update ist zu prüfen, dass dieses auch kompatibel mit der eingesetzten Joomla-Installation ist.<br>Es wird dringend empfohlen, vor dem Updates ein <strong>Backup</strong> der Installation zu erstellen."
 COM_INSTALLER_MSG_WARNINGS_UPLOADBIGGERTHANPOST="PHP-Hochladegröße größer als POST-Größe"
 COM_INSTALLER_MSG_WARNINGS_UPLOADBIGGERTHANPOSTDESC="Der Wert für die „upload_max_filesize“ in der „php.ini“-Datei ist größer als die „post_max_size“-Variable. Der Wert der „post_max_size“-Variable hat Priorität und wird zu große Anfragen blockieren. Dies ist meist auf eine Fehleinstellung des Servers beim Erhöhen der Dateihochladegröße zurückzuführen. Es sollte daher der Wert von „upload_max_filesize“ mindestens auf das Niveau der „post_max_size“-Variable erhöht werden oder umgekehrt."
 COM_INSTALLER_MSG_WARNINGS_UPLOADFILETOOBIG="Die ausgewählte Datei kann nicht hochgeladen werden, da sie größer als die maximale Dateigröße ist."
 COM_INSTALLER_N_EXTENSIONS_PUBLISHED="%d Erweiterungen wurden aktiviert!"
 COM_INSTALLER_N_EXTENSIONS_PUBLISHED_1="%d Erweiterung wurde aktiviert!"
 COM_INSTALLER_N_EXTENSIONS_UNPUBLISHED="%d Erweiterungen wurden deaktiviert!"
 COM_INSTALLER_N_EXTENSIONS_UNPUBLISHED_1="%d Erweiterung wurde deaktiviert!"
-COM_INSTALLER_N_UPDATESITES_PUBLISHED="%d Aktualisierungsquellen wurden aktiviert!"
-COM_INSTALLER_N_UPDATESITES_PUBLISHED_1="%d Aktualisierungsquelle wurde aktiviert!"
-COM_INSTALLER_N_UPDATESITES_UNPUBLISHED="%d Aktualisierungsquellen wurden deaktiviert!"
-COM_INSTALLER_N_UPDATESITES_UNPUBLISHED_1="%d Aktualisierungsquelle wurde deaktiviert!"
+COM_INSTALLER_N_UPDATESITES_PUBLISHED="%d Update-Quellen wurden aktiviert!"
+COM_INSTALLER_N_UPDATESITES_PUBLISHED_1="%d Update-Quelle wurde aktiviert!"
+COM_INSTALLER_N_UPDATESITES_UNPUBLISHED="%d Update-Quellen wurden deaktiviert!"
```

Comment:
"Eine Update-Quelle"

Reply:
> "Eine Update-Quelle"

Ja, aber wir schreiben da bisher bei Einzahl wohl noch %d

Reply:
Scheint nicht durchgängig zu sein

Reply:
Sollte evtl. in einem eigenen PR gemacht werden.